modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFolderItemLocalServiceTest.java

```diff
@@ -264,6 +279,51 @@ public void testDeleteObjectFolderItemByObjectFolderId() {
 						_objectFolderA.getObjectFolderId())));
 	}
 
+	@Test
+	public void testGetObjectFolderItemByObjectFolderId() throws Exception {
+		String originalName = PrincipalThreadLocal.getName();
+		PermissionChecker originalPermissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		try {
+			User user = UserTestUtil.addUser();
+
+			PermissionThreadLocal.setPermissionChecker(
+				PermissionCheckerFactoryUtil.create(user));
+			PrincipalThreadLocal.setName(user.getUserId());
+
+			Role role = RoleTestUtil.addRole(RoleConstants.TYPE_REGULAR);
+
+			_resourcePermissionLocalService.setResourcePermissions(
+				TestPropsValues.getCompanyId(), ObjectFolder.class.getName(),
+				ResourceConstants.SCOPE_COMPANY,
+				String.valueOf(TestPropsValues.getCompanyId()),
+				role.getRoleId(), new String[] {ActionKeys.VIEW});
+
+			_userLocalService.addRoleUser(role.getRoleId(), user);
+
+			ObjectFolder objectFolder =
+				_objectFolderLocalService.getUncategorizedObjectFolder(
```

Comment:
Do uncategorized ObjectFolder has some ObjectItens? I don't see any ObjectDefinition linked to this folder either in this test and in set up.

Reply:
System objectDefinitions are into uncategorized folder